src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,24 +1,174 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 /**
  * <p>Реалізувати свою HashMap, а саме методи `put(K key, V value)`, `getValue()` та `getSize()`.
  * Дотриматися основних вимог щодо реалізації мапи (initial capacity, load factor, resize...)
  * За бажанням можна реалізувати інші методи інтрефейсу Map.</p>
  */
 public class MyHashMap<K, V> implements MyMap<K, V> {
 
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int DEFAULT_BUCKETS = 16;
+
+    private Node[] table;
+    private int buckets;
+    private int capacity;
```

Comment:
what is the difference between buckets and capacity?
let's remove `buckets`  and `capacity`
try using only size and table.length

Reply:
buckets == table.length
capacity - number of used buckets

Reply:
done